app/src/features/datasets/DatasetPage.tsx

```diff
@@ -77,6 +87,14 @@ const DatasetPage: React.FC = () => {
     })
   );
 
+  useDataLoaderError(fileDataLoader, () => {
+    return {
+      dialogTitle: 'Error generating file',
```

Comment:
super nit: capitalize the words in the title

Reply:
done